scripts/tb.sh

```diff
@@ -0,0 +1,290 @@
+#!/bin/bash
+
+set -Eeu
+
+print_help() {
+    cat << EOF
+Usage:
+    ./tb.sh [OPTION]... {modify|reset|menuconfig|build|deploy}
+    1.) ./tb.sh modify recipe
+    2.) ./tb.sh reset recipe
+    3.) ./tb.sh menuconfig
+    4.) ./tb.sh build recipe
+    5.) ./tb.sh deploy recipe destination
+
+    1.) Fetches source code for the recipe
+    2.) Removes recipe source code from disk
+    3.) Opens Linux menuconfig
+    4.) Builds recipe from fetched source code.
+    5.) Copies built recipe to destination
+
+    OPTION:
+        -h|--help               - Print this help
+        -v|--verbose            - Verbose output
+
+    recipe:
+        skl                     - Secure Kernel Loader
+        linux-tb                - Linux kernel
+        grub                    - GRUB legacy
+        grub-efi                - GRUB EFI
+        tb-minimal-image        - Complete image with all components. Can only
+                                  be build or deployed
+
+    destination:
+        Path to target root '/' directory with boot partition mounted at
+        '/boot'. Uses same syntax as rsync e.g.:
+
+        /mnt                    - locally mounted TrenchBoot rootfs partition at
+                                  '/mnt' with boot partition mounted
+                                  at '/mnt/boot'
+        root@192.168.4.10:/     - remote machine running TrenchBoot
+        root@192.168.4.10:/mnt  - remote machine with TrenchBoot rootfs
+                                  partition mounted at '/mnt' and boot partition
+                                  mounted at '/mnt/boot'
+EOF
+}
+
+cleanup() {
+    popd &>/dev/null || exit 1
+}
+
+usage_error() {
+    echo "$1"
+    print_help
+    exit 1
+}
+
+error_msg() {
+    echo "$1"
+    exit 1
+}
+
+parse_args() {
+    local positional_args=()
+    ACTION=
+    RECIPE_ARG=
+    DESTINATION_ARG=
+
+    while [ $# -gt 0 ]; do
+        case $1 in
+            -h|--help)
+                print_help
+                exit 0
+                ;;
+            -v|--verbose)
+                set -x
+                ;;
+            -*)
+                usage_error "Unknown option"
```

Comment:
```suggestion
                usage_error "Unknown option '$1'"
```

Reply:
fixed in [bb92812](https://github.com/3mdeb/meta-trenchboot/pull/29/commits/bb92812810bdd5aefbfb186944295f2faa4359f5)